tests/e2e/e2e.go

```diff
@@ -156,7 +156,7 @@ func (te *TestEnvironment) StartCluster() error {
 	switch te.clusterType {
 	case StandAlone:
 		tests.Outf("{{magenta}}starting network-runner with %q{{/}}\n", te.avalancheGoExecPath)
-		ctx, cancel := context.WithTimeout(context.Background(), 15*time.Second)
+		ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
```

Comment:
do we have any theories on why this starting to failing or has it been a flake? Is there anything else happening in background/parallel? ~16s seems like a really long time.

Reply:
I think that the ANR has a number of internal sleeps currently that take a long time 😞. But will let @marun comment